### PR TITLE
Allow for config files in local directory

### PR DESCRIPTION
**Pull request by SebastianM-C**
Document the changes from https://github.com/JuliaPlots/AbstractPlotting.jl/pull/125 and describe a bit more the config file.

### CODE REVIEW

File: docs/src/theming.md [hunk 1]
```diff
@@ -11,4 +18,4 @@ Attributes(
     linewidth = 3
 )
 ```
-To start Makie always with this theme!
+then Makie will with this theme.
```
Comment (asinghvi17) [inline docs/src/theming.md]: Maybe reword to something like:

```then Makie will start with this theme.```

?

Comment (SebastianM-C) [reply docs/src/theming.md]: yeah, I don't know how that appeared 😆

**Comment by asinghvi17** [comment]
Lovely, thanks!
